alpha/alpha-core/src/main/java/org/apache/servicecomb/pack/alpha/core/fsm/repository/model/GlobalTransaction.java

```diff
@@ -36,11 +36,12 @@
   private Date beginTime;
   @JsonFormat(shape = JsonFormat.Shape.NUMBER)
   private Date endTime;
-  private SagaActorState state;
+  private String state;
```

Comment:
Is there any reason that we change to String?

Reply:
> Is there any reason that we change to String?

Because SagaActorState is defined in alpha-fsm, this causes alpha-fsm and alpha-core cross-references